docs/wikilink_demo.mdx

```diff
@@ -0,0 +1,19 @@
+---
+slug: /wikilink_demo
```

Comment:
The slug will default to the file's location relative to `/docs`, I think this is redundant.